client/Windows/wf_cliprdr.c

```diff
@@ -2046,7 +2046,7 @@ static BOOL wf_cliprdr_process_filename(wfClipboard* clipboard,
 	if (!wf_cliprdr_add_to_file_arrays(clipboard, wFileName, pathLen))
 		return FALSE;
 
-	if ((clipboard->fileDescriptor[clipboard->nFiles - 1]->dwFileAttributes &
+	if ((clipboard->fileDescriptor[clipboard->nFiles - 1]->dwFileAttributes&
```

Comment:
Formatting is screwed here

Reply:
@mfleisz astyle accepts, although you're right, it looks weired.